scripts/setup.sh

```diff
@@ -43,6 +43,11 @@ function maybeInstallDependencies {
       shift
       ;;
 
+      --suite=*)
```

Comment:
Sharding doesn't throws the error because it's called later after the setup.sh is called. It never reaches setup.sh script. But suite is passed to setup. Actually the error is raised by line 52 on setup.sh.